config/jobs/image-pushing/k8s-staging-cloud-provider-gcp.yaml

```diff
@@ -29,3 +29,35 @@ postsubmits:
               - --scratch-bucket=gs://k8s-staging-cloud-provider-gcp-gcb
               - --env-passthrough=PULL_BASE_REF
               - .
+  kubernetes-sigs/gcp-compute-persistent-disk-csi-driver:
+    - name: post-gcp-compute-persistent-disk-csi-driver-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: provider-gcp-filestore-csi-driver
```

Comment:
I think you want the pd csi testgrids at https://testgrid.k8s.io/provider-gcp-compute-persistent-disk-csi-driver?

Reply:
did not know that one ( ERR_TOO_MANY_DASHBOARDS 😂  )

updated